test/utils/crud.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/klog/v2"
+	ctrlClient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type K8sCRUD struct {
+	c ctrlClient.Client
+}
+
+func NewK8sCRUD(c ctrlClient.Client) *K8sCRUD {
+	return &K8sCRUD{c: c}
+}
+
+func (crud *K8sCRUD) GetClient() ctrlClient.Client {
```

Comment:
why do we need this method?

Reply:
Updated.

Reply:
Yes you're right, I thought we need getter. Updated! Thanks for calling out.